fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/api/LoanTransactionsApiResource.java

```diff
@@ -456,6 +460,8 @@ private String executeTransaction(final Long loanId, final String loanExternalId
             commandRequest = builder.loanForeclosure(resolvedLoanId).build();
         } else if (CommandParameterUtil.is(commandParam, "creditBalanceRefund")) {
             commandRequest = builder.creditBalanceRefund(resolvedLoanId).build();
+        } else if (CommandParameterUtil.is(commandParam, "charge-off")) {
```

Comment:
Do you think you could extract the command name to a constant? Don't need to do anything with the existing ones. Just wanna make sure we show guideline for the future commands.